includes/managers/icons.php

```diff
@@ -475,40 +580,42 @@ public function register_admin_settings( Settings $settings ) {
 						'' => esc_html__( 'No', 'elementor' ),
 						'yes' => esc_html__( 'Yes', 'elementor' ),
 					],
-					'desc' => esc_html__( 'Font Awesome 4 support script (shim.js) is a script that makes sure all previously selected Font Awesome 4 icons are displayed correctly while using Font Awesome 5 library.', 'elementor' ),
+					'desc' => esc_html__( 'Font Awesome 4 support script (shim.js) is a script that makes sure all previously selected Font Awesome 4 icons are displayed correctly while using later Font Awesome versions.', 'elementor' ),
 				],
 			]
 		);
 	}
 
 	public function register_admin_tools_settings( Tools $settings ) {
-		$settings->add_tab( 'fontawesome4_migration', [ 'label' => esc_html__( 'Font Awesome Upgrade', 'elementor' ) ] );
+		$settings->add_tab( 'fontawesome_migration', [ 'label' => esc_html__( 'Font Awesome', 'elementor' ) ] );
 
-		$settings->add_section( 'fontawesome4_migration', 'fontawesome4_migration', [
+		$settings->add_section( 'fontawesome_migration', 'fontawesome_migration', [
 			'callback' => function() {
 				echo '<h2>' . esc_html__( 'Font Awesome Upgrade', 'elementor' ) . '</h2>';
 				echo '<p>' . // PHPCS - Plain Text
-				esc_html__( 'Access 1,500+ amazing Font Awesome 5 icons and enjoy faster performance and design flexibility.', 'elementor' ) . '<br>' . // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-				esc_html__( 'By upgrading, whenever you edit a page containing a Font Awesome 4 icon, Elementor will convert it to the new Font Awesome 5 icon.', 'elementor' ) . // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				/* translators: %s: Version number. */
+				sprintf( esc_html__( 'Access 1,700+ amazing icons, faster performance, and design flexibility with Font Awesome v%s.', 'elementor' ), self::get_current_fa_version() ) . // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
```suggestion
				sprintf( esc_html__( 'Access 1,700+ amazing icons, faster performance and design flexibility with Font Awesome v%s.', 'elementor' ), self::get_current_fa_version() ) . // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Reply:
The text comes from content writers :)

Reply:
Isn't it a grammar mistake?

Reply:
No, depends on one's opinion on [the Serial Comma](https://en.wikipedia.org/wiki/Serial_comma). In a strict following of the AP style guide, you would need to leave it out, but in almost every other style guide, you would put it in.